database/migrations/2017_07_28_175306_create_users_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('users', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('full_name');
+            $table->string('email')->unique();
+            $table->string('password');
+            $table->date('birthday')->nullable();
+            $table->boolean('gender')->nullable();
```

Comment:
you should use comment statement to define value. for example: 
$table->tinyInteger('send_type')->default(1)
            ->comment = 'IMMEDIATELY = 1; APPOINTMENT = 2; PAUSE = 3; END = 4';